packages/e2e/cypress/integration/e2e/Canvas/canvas.cy.ts

```diff
@@ -0,0 +1,115 @@
+import { BROWSER_PX_DEVIATION, WheelDirection } from "../constants"
+import { coordinatesFromStringPX, zoomFromMatrix } from "../helpers"
+import { canvasModel } from "./Canvas.model"
+import { CONTEXT } from "./context"
+import { checkCanvasPosition } from "./helpers"
+
+context(CONTEXT, () => {
+  beforeEach(canvasModel.open)
+
+  const checkCanvasOriginPosition = (x: number, y: number) =>
+    canvasModel
+      .canvasPositionOrigin()
+      .then(coordinatesFromStringPX)
+      .then(([xCoord, yCoord]) => {
+        expect(Number(xCoord)).to.be.closeTo(x, BROWSER_PX_DEVIATION)
+        expect(Number(yCoord)).to.be.closeTo(y, BROWSER_PX_DEVIATION)
+      })
+
+  describe("Initializing", () => {
+    it("Should match default position", () => {
+      checkCanvasPosition(15, -35)
```

Comment:
Magical. numbers

Reply:
ok, i've looked through tests and replaced a part of numbers to constants that has real meaningful meaning and is reused many times (i. e. nodes coords, connectors coords), but some other magic numbers have no it's meaningful meaning and is used 'in place' in tests, so, i suppose that part of numbers is redundant to be moved to constants